docs/tools/CLI/cli-setup/cli-reqs.md

```diff
@@ -228,6 +228,12 @@ Mbed CLI supports a setting for each toolchain path:
 | IAR EWARM Compiler | `C:/Program Files/IAR Systems/Embedded Workbench 8.2/arm/bin/iccarm.exe` | `IAR_PATH` | `C:/Program Files/IAR Systems/Embedded Workbench 8.2/arm`|
 | GCC Arm Embedded Compiler | `/usr/bin/arm-none-eabi-gcc` | `GCC_ARM_PATH` | `/usr/bin`|
 
+##### Using Arm Compiler 6 from Mbed Studio
```

Comment:
Is it weird that this is in the Mbed OS docs? Studio documentation is now 100% somewhere else: https://os.mbed.com/docs/mbed-studio/

Reply:
This is in the Mbed CLI docs. The point of this change is to enable the use of Mbed Studio's version of Arm Compiler 6 when compiling with Mbed CLI. I think this is the most appropriate place, however if there is a better place just let me know.

Reply:
Sorry, your wording confused me. "From Mbed Studio" - I took that to mean that you're using Mbed Studio, not that you're using something that belongs to Mbed Studio

Reply:
So are you saying that if I'm using Mbed CLI, I need to install Mbed Studio or something?